test_module.py

```diff
@@ -0,0 +1,58 @@
+# this module is a test module to allow for testing of the program without the need for the gpt3.5 model
```

Comment:
Recommend we change this to Unit Testing and start implementing those!

Reply:
this module was used to test the kobold module befor i implimented it. this entire file can be safely deleted